src/js/components/PanelCell/PanelCell.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { getClassName } from '../../utils/helpers'
+
+class PanelCell extends PureComponent {
+  renderChildrenHeader = (children) => <h4 className="Panel-title">{ children }</h4>
```

Comment:
So in this case the parent `Panel-cell--header` is applying style to the text element, so `Panel-title` isn't needed.